.github/CODEOWNERS

```
@@ -0,0 +1,14 @@
+# GitHub CODEOWNERS. See:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# Remember, the last match takes precedence.
+
+# Replace the Starcraft team with your appropriate code owners.
+
+# Global owner, used as a last resort.
+*       @mr-cal
+
+# Documentation owners
+/docs/  @medubelko @mr-cal
+
+# Finally, all CODEOWNERS changes need to be approved by The Man, Himself.
+/.github/CODEOWNERS     @sergiusens
```

You can give env_injector to me: 
```suggestion
/.github/CODEOWNERS     @sergiusens

/snapcraft/extensions/env_injector.py @locnnil @farshidtz 
/tests/spread/extensions/env-injector @locnnil @farshidtz 
```